### PR TITLE
Various text changes.

### PR DESCRIPTION
**Pull request by sjwaight**
Hit me up if you want to discuss anything contained in the PR.

### CODE REVIEW

File: static-web-apps/tutorial.md [hunk 1]
```diff
@@ -141,67 +136,66 @@ title: Create an Azure Static Web App
...
-![Enter GitHub information when creating SWA](media/swa-github.png)
+![Enter GitHub information when creating SWA](images/swa-github.png)
 
 * Click on `Review + Create` and then on `Create`.
 
-After a few minutes, your static web app will be created on Azure and your website deployed.
+After a few minutes, your Static Web App will be created on Azure and your website deployed.
 
-Once the resource is created, `pull` your code as a few files have been added to your GitHub repo by Azure.
+Once the Azure Resources are created, you should `pull` your repository in VSCode as a few files have been added to your GitHub repo by Azure.
```
Comment (tagazok) [inline static-web-apps/tutorial.md:179]: It only creates one resource, the Static Web App one

File: static-web-apps/tutorial.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ title: Objectives & Prerequesites
 
 In this woskhop, you will learn how to create a full application with a `frontend`, a `serverless backend`, `user authentication` and a `database` to store your data.
 
-Don't worry if you are not familiar with Microsoft Azure. This workshop will walk you through some of the most important services if you are a web developer. We will see how to use `Cosmos DB` with the `MongoDB` API, `Azure Static Web Apps` to host your client application and your `Azure Functions` and `GitHub Actions` to automate your app deployment!
+Don't worry if you are not familiar with Microsoft Azure. This workshop will walk you through some of the most important services if you are a web developer. We will see how to use `Cosmos DB` with the `MongoDB` API, `Azure Static Web Apps` to host your client application and `Azure Functions` for your backend and `GitHub Actions` to automate your app deployment!
```
Comment (tagazok) [inline static-web-apps/tutorial.md:15]: Technically, SWA hosts both the frontend and the functions. There is no Azure Function resource created on Azure.

File: static-web-apps/tutorial.md [hunk 3]
```diff
@@ -435,13 +433,13 @@ Eg. `?post_login_redirect_uri=/index.html`
...
 <div>
-If you are building a React app, go <a href="https://docs.microsoft.com/en-us/learn/modules/publish-static-web-app-authentication/?WT.mc_id=javascript-17844-cxa" target="_blank">check the MSLearn module that will show you how to do it.</a>
+If you are building a React app, go <a href="https://docs.microsoft.com/learn/modules/publish-static-web-app-authentication/?WT.mc_id=javascript-17844-cxa" target="_blank">check the Microsoft Learn module</a> that will show you how to do it.
 </div>
 </div>
 
 ### Getting user information
 
-Once your user is authenticated, you can retrieve the associated information by fetching the url `/.auth/me`. This will return a json containing a clientPrincipal object. If the object is null, the user is not authenticated. Otherwize, the object contains data like the provider, the roles and the username.
+Once your user is authenticated, you can retrieve the user's information by fetching the url `/.auth/me`. This will return JSON containing a clientPrincipal object. If the object is null, the user is not authenticated. Otherwise, the object contains data like the provider, the roles and the username.
```
[the remark this answers is not in the document]
Comment (sjwaight) [reply static-web-apps/tutorial.md:442]: Some people might write "a blob of JSON" or "a block of JSON" or "some JSON".

File: static-web-apps/tutorial.md (hunk 2 above)
Comment (tagazok) [reply static-web-apps/tutorial.md:15]: oh ok I see :)

File: static-web-apps/tutorial.md (hunk 1 above)
Comment (sjwaight) [reply static-web-apps/tutorial.md:179]: OK, update the first component to "Once the resource is created..."

**Comment by sjwaight** [comment]
Let me know if you want me to fix and submit another PR, or if you want to fix based on the comments.

**Pull request merged**